core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -123,7 +123,6 @@ public boolean hasOwner() {
    * @return the owner
    */
   public String getOwner() {
-    Preconditions.checkState(hasOwner(), PreconditionMessage.MUST_SET_OWNER);
```

Comment:
and without the serialization of SetAttributeOptions that do not have these options set fails